DOCS-12212 Clarifying limitations (#497)
mulesoft/docs-runtime-fabric: modules/ROOT/pages/limitations-self.adoc

| Replicas Per Node | The maximum number of replicas that can be deployed per node is 40.

Run no more than 20 - 25 replicas per core, up to a maximum of 40 replicas per node, to allow core sharing across replicas when needed for bursting. This ensures the Runtime Fabric’s internal components that run on each worker node are not overloaded by too many replicas.

> | Associated Environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that's counted as three environment associations. 

Author: ```suggestion
| Associated Environments per Runtime Fabric | You can associate a Runtime Fabric with up to 100 environments in any Business Group. For example, if you associate a development and a production environment with Org A and a dev environment with Org B, that is three environment associations. 
```
